tomviz/OperatorPython.cxx

```diff
@@ -227,27 +264,75 @@ bool OperatorPython::applyTransform(vtkDataObject* data)
     return false;
   }
 
-  // TODO - check if there are results but no settings for them in
-  // the output from the script.
-
   // Look for additional outputs from the filter returned in a dictionary
   PyObject* outputDict = result.GetPointer();
   if (PyDict_Check(outputDict)) {
-    for (int i = 0; i < numberOfResults(); ++i) {
-      OperatorResult* operatorResult = resultAt(i);
-      std::string resultName = operatorResult->name().toStdString();
-      PyObject* pyDataObject =
-        PyDict_GetItemString(outputDict, resultName.c_str());
-      if (pyDataObject) {
-        vtkObjectBase* vtkobject =
-          vtkPythonUtil::GetPointerFromObject(pyDataObject, "vtkDataObject");
-        if (vtkobject) {
-          setResult(i, vtkDataObject::SafeDownCast(vtkobject));
+
+    // Results (tables, etc.)
+    for (int i = 0; i < m_resultNames.size(); ++i) {
+      const char* name = m_resultNames[i].toStdString().c_str();
```

Comment:
I think you mean toLatin1 here anyway, and using

  const char* name = m_resultNames[i].toLatin1().data(); 

Gives me the results I would expect locally.